### ISSUE TITLE
dynamic igemm not applicable for many cases

### ISSUE THREAD

**Issue by asleepzzz** [ROCm/MIOpen#648]
dynamic igemm can't pass 
just make test_conv_igemm_dynamic_small

**Comment by carlushuang** [comment]
this test `test_conv_igemm_dynamic_small` is for non-xdlops and should not be run in xdlops environment.

**Comment by carlushuang** [comment]
most resent CI pass all is http://micimaster.amd.com/blue/organizations/jenkins/MLLibs%2FMIOpen/detail/develop/168/pipeline

**Comment by daniellowell** [comment]
@asleepzzz Add long logs as attachments.

**Comment by atamazov** [comment]
@carlushuang 
> most resent CI pass all is http://micimaster.amd.com/blue/organizations/jenkins/MLLibs%2FMIOpen/detail/develop/168/pipeline

Is it related to this issue?

**Comment by atamazov** [comment]
[off-topic] 
> @asleepzzz Add long logs as attachments.

You can also use collapsible sections:
````
<details><summary><b><i>HEADLINE</i></b></summary>

```
INSERT LOG CONTENTS HERE
```
</details>
````

**Comment by asleepzzz** [comment]
http://micimaster.amd.com/blue/organizations/jenkins/MLLibs%2FMIOpen/detail/fix_v4r1bwd_tunning/5/pipeline/

**Comment by asleepzzz** [comment]
<details><summary><b><i>HEADLINE</i></b></summary>

```
Test project /var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/build/test

      Start 11: test_conv2d

      Start 17: test_dropout

 1/57 Test #17: test_dropout .....................***Not Run (Disabled)   0.00 sec

      Start 28: test_lrn_test

 2/57 Test #28: test_lrn_test ....................***Not Run (Disabled)   0.00 sec

      Start 36: test_pooling2d

 3/57 Test #36: test_pooling2d ...................***Not Run (Disabled)   0.00 sec

      Start 37: test_pooling3d

 4/57 Test #37: test_pooling3d ...................***Not Run (Disabled)   0.00 sec

      Start 42: test_soft_max

 5/57 Test #42: test_soft_max ....................***Not Run (Disabled)   0.00 sec

      Start 57: test_conv_igemm_dynamic_small

      Start  1: test_activation

 6/57 Test  #1: test_activation ..................***Not Run (Disabled)   0.00 sec

      Start  2: test_bn_3d_peract_test

 7/57 Test  #2: test_bn_3d_peract_test ...........***Not Run (Disabled)   0.00 sec

      Start  3: test_bn_3d_spatial_test

 8/57 Test  #3: test_bn_3d_spatial_test ..........***Not Run (Disabled)   0.00 sec

      Start  4: test_bn_aux

 9/57 Test  #4: test_bn_aux ......................***Not Run (Disabled)   0.00 sec

      Start  5: test_bn_peract_test

10/57 Test  #5: test_bn_peract_test ..............***Not Run (Disabled)   0.00 sec

      Start  6: test_bn_spatial_test

11/57 Test  #6: test_bn_spatial_test .............***Not Run (Disabled)   0.00 sec

      Start  7: test_cache

12/57 Test  #7: test_cache .......................***Not Run (Disabled)   0.00 sec

      Start  8: test_cba_inference

13/57 Test  #8: test_cba_inference ...............***Not Run (Disabled)   0.00 sec

      Start  9: test_cbna_inference

14/57 Test  #9: test_cbna_inference ..............***Not Run (Disabled)   0.00 sec

      Start 10: test_check_numerics_test

      Start 12: test_conv2d_bias

15/57 Test #12: test_conv2d_bias .................***Not Run (Disabled)   0.00 sec

      Start 13: test_conv3d

16/57 Test #13: test_conv3d ......................***Not Run (Disabled)   0.00 sec

      Start 14: test_conv3d_bias

17/57 Test #14: test_conv3d_bias .................***Not Run (Disabled)   0.00 sec

      Start 15: test_ctc

18/57 Test #15: test_ctc .........................***Not Run (Disabled)   0.00 sec

      Start 16: test_custom_allocator

19/57 Test #16: test_custom_allocator ............***Not Run (Disabled)   0.00 sec

      Start 18: test_embed_sqlite

20/57 Test #18: test_embed_sqlite ................***Not Run (Disabled)   0.00 sec

      Start 19: test_find_db

21/57 Test #19: test_find_db .....................***Not Run (Disabled)   0.00 sec

      Start 20: test_fusion_aux

22/57 Test #20: test_fusion_aux ..................***Not Run (Disabled)   0.00 sec

      Start 21: test_gru

23/57 Test #21: test_gru .........................***Not Run (Disabled)   0.00 sec

      Start 22: test_gru_dropout

24/57 Test #22: test_gru_dropout .................***Not Run (Disabled)   0.00 sec

      Start 23: test_handle_test

25/57 Test #23: test_handle_test .................***Not Run (Disabled)   0.00 sec

      Start 24: test_immed_conv2d

26/57 Test #24: test_immed_conv2d ................   Passed    1.30 sec

      Start 25: test_immed_conv3d

27/57 Test #25: test_immed_conv3d ................***Not Run (Disabled)   0.00 sec

28/57 Test #10: test_check_numerics_test .........   Passed    5.41 sec

      Start 26: test_include_inliner

29/57 Test #26: test_include_inliner .............***Not Run (Disabled)   0.00 sec

      Start 27: test_kernel_build_params

30/57 Test #27: test_kernel_build_params .........***Not Run (Disabled)   0.00 sec

      Start 29: test_lstm

31/57 Test #29: test_lstm ........................***Not Run (Disabled)   0.00 sec

      Start 30: test_lstm_dropout

32/57 Test #30: test_lstm_dropout ................***Not Run (Disabled)   0.00 sec

      Start 31: test_main

33/57 Test #31: test_main ........................***Not Run (Disabled)   0.00 sec

      Start 32: test_mdgraph

34/57 Test #32: test_mdgraph .....................***Not Run (Disabled)   0.00 sec

      Start 33: test_na_inference

35/57 Test #33: test_na_inference ................***Not Run (Disabled)   0.00 sec

      Start 34: test_na_train

36/57 Test #34: test_na_train ....................***Not Run (Disabled)   0.00 sec

      Start 38: test_reduce_test

37/57 Test #38: test_reduce_test .................***Not Run (Disabled)   0.00 sec

      Start 39: test_rnn_vanilla

38/57 Test #39: test_rnn_vanilla .................***Not Run (Disabled)   0.00 sec

      Start 40: test_rnn_vanilla_dropout

39/57 Test #40: test_rnn_vanilla_dropout .........***Not Run (Disabled)   0.00 sec

      Start 41: test_sequences

40/57 Test #41: test_sequences ...................***Not Run (Disabled)   0.00 sec

      Start 43: test_solver

41/57 Test #43: test_solver ......................***Not Run (Disabled)   0.00 sec

      Start 45: test_tensor_cast

42/57 Test #45: test_tensor_cast .................***Not Run (Disabled)   0.00 sec

      Start 46: test_tensor_copy

      Start 47: test_tensor_ops

43/57 Test #47: test_tensor_ops ..................***Not Run (Disabled)   0.00 sec

      Start 48: test_tensor_scale

44/57 Test #48: test_tensor_scale ................***Not Run (Disabled)   0.00 sec

      Start 49: test_tensor_set

45/57 Test #49: test_tensor_set ..................   Passed    6.11 sec

      Start 50: test_tensor_test

46/57 Test #50: test_tensor_test .................***Not Run (Disabled)   0.00 sec

47/57 Test #46: test_tensor_copy .................   Passed    7.12 sec

      Start 51: test_tensor_trans

48/57 Test #51: test_tensor_trans ................***Not Run (Disabled)   0.00 sec

      Start 52: test_tensor_transform

49/57 Test #52: test_tensor_transform ............***Not Run (Disabled)   0.00 sec

      Start 53: test_tensor_vec

      Start 54: test_test_errors

50/57 Test #54: test_test_errors .................***Not Run (Disabled)   0.00 sec

      Start 55: test_type_name

51/57 Test #55: test_type_name ...................***Not Run (Disabled)   0.00 sec

      Start 56: test_w_supertensor

52/57 Test #56: test_w_supertensor ...............***Not Run (Disabled)   0.00 sec

53/57 Test #57: test_conv_igemm_dynamic_small ....***Failed   16.15 sec

[  0%] Built target sqlite_memvfs

[  3%] Built target addkernels

[100%] Built target MIOpen

[100%] Built target test_conv2d

Scanning dependencies of target test_conv_igemm_dynamic_small

MIOpen(HIP): Info [GetFindModeValueImpl] MIOPEN_FIND_MODE = HYBRID(3)

MIOpen(HIP): Info [Handle] stream: 0xaebadf0, device_id: 2

MIOpen(HIP): Info [BackwardDataGetWorkSpaceSize] 

MIOpen(HIP): Info [HipCompilerVersionImpl] 3.0.19502

MIOpen(HIP): Info [AmdRocmMetadataVersionDetect] ROCm MD version AMDHSA_COv2_COv3, MIOpen version 2.10.0.8301-71981a35

MIOpen(HIP): Info [GetBackwardSolutions] 

MIOpen(HIP): Info [Measure] Db::Prefetch time: 114.662 ms

MIOpen(HIP): Info [GetEnvFindOnlySolverImpl] 62

MIOpen(HIP): Info [ForwardGetWorkSpaceSize] 

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [BackwardWeightsGetWorkSpaceSize] 

MIOpen(HIP): Info [GetWrwSolutions] 

/var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/build/bin/test_conv2d --float --cmode conv --pmode default --group-count 1 --disable-backward-data --disable-backward-weights --input 16, 16, 56, 56 --weights 64, 16, 1, 1 --pads_strides_dilations 0 0 1 1 1 1 --trans_output_pads 0 0 

MIOpen(HIP): Info [ForwardGetWorkSpaceSize] 

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [FindConvFwdAlgorithm] requestAlgoCount = 1, workspace = 0

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [TryLoad] Find-db regenerating.

/var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/build/bin/test_conv2d --float --cmode conv --pmode default --group-count 1 --disable-backward-data --disable-backward-weights --input 16, 16, 56, 56 --weights 64, 16, 1, 1 --pads_strides_dilations 0 0 1 1 1 1 --trans_output_pads 0 0 

FAILED: /var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/src/ocl/convolutionocl.cpp:783: Forward Convolution cannot be executed due to incorrect params

Forward convolution: 

Input tensor: 16, 16, 56, 56

Weights tensor: 64, 16, 1, 1

Output tensor: 

Filter: conv2d, miopenConvolution, miopenPaddingDefault, {0, 0}, {1, 1}, {1, 1}, 

MIOpen(HIP): Info [GetFindModeValueImpl] MIOPEN_FIND_MODE = HYBRID(3)

MIOpen(HIP): Info [Handle] stream: 0x3c11fe0, device_id: 3

MIOpen(HIP): Info [BackwardDataGetWorkSpaceSize] 

MIOpen(HIP): Info [HipCompilerVersionImpl] 3.0.19502

MIOpen(HIP): Info [AmdRocmMetadataVersionDetect] ROCm MD version AMDHSA_COv2_COv3, MIOpen version 2.10.0.8301-71981a35

MIOpen(HIP): Info [GetBackwardSolutions] 

MIOpen(HIP): Info [Measure] Db::Prefetch time: 112.943 ms

MIOpen(HIP): Info [GetEnvFindOnlySolverImpl] 62

MIOpen(HIP): Info [ForwardGetWorkSpaceSize] 

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [BackwardWeightsGetWorkSpaceSize] 

MIOpen(HIP): Info [GetWrwSolutions] 

/var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/build/bin/test_conv2d --float --cmode conv --pmode default --group-count 1 --disable-backward-data --disable-backward-weights --input 16, 64, 34, 34 --weights 64, 64, 3, 3 --pads_strides_dilations 0 0 1 1 1 1 --trans_output_pads 0 0 

MIOpen(HIP): Info [ForwardGetWorkSpaceSize] 

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [FindConvFwdAlgorithm] requestAlgoCount = 1, workspace = 0

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [TryLoad] Find-db regenerating.

/var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/build/bin/test_conv2d --float --cmode conv --pmode default --group-count 1 --disable-backward-data --disable-backward-weights --input 16, 64, 34, 34 --weights 64, 64, 3, 3 --pads_strides_dilations 0 0 1 1 1 1 --trans_output_pads 0 0 

FAILED: /var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/src/ocl/convolutionocl.cpp:783: Forward Convolution cannot be executed due to incorrect params

Forward convolution: 

Input tensor: 16, 64, 34, 34

Weights tensor: 64, 64, 3, 3

Output tensor: 

Filter: conv2d, miopenConvolution, miopenPaddingDefault, {0, 0}, {1, 1}, {1, 1}, 

MIOpen(HIP): Info [GetFindModeValueImpl] MIOPEN_FIND_MODE = HYBRID(3)

MIOpen(HIP): Info [Handle] stream: 0xa967e30, device_id: 3

MIOpen(HIP): Info [BackwardDataGetWorkSpaceSize] 

MIOpen(HIP): Info [HipCompilerVersionImpl] 3.0.19502

MIOpen(HIP): Info [AmdRocmMetadataVersionDetect] ROCm MD version AMDHSA_COv2_COv3, MIOpen version 2.10.0.8301-71981a35

MIOpen(HIP): Info [GetBackwardSolutions] 

MIOpen(HIP): Info [Measure] Db::Prefetch time: 139.963 ms

MIOpen(HIP): Info [GetEnvFindOnlySolverImpl] 62

MIOpen(HIP): Info [ForwardGetWorkSpaceSize] 

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [BackwardWeightsGetWorkSpaceSize] 

MIOpen(HIP): Info [GetWrwSolutions] 

/var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/build/bin/test_conv2d --float --cmode conv --pmode default --group-count 1 --disable-backward-data --disable-backward-weights --input 32, 32, 17, 17 --weights 32, 32, 1, 7 --pads_strides_dilations 0 3 1 1 1 1 --trans_output_pads 0 0 

MIOpen(HIP): Info [ForwardGetWorkSpaceSize] 

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [FindConvFwdAlgorithm] requestAlgoCount = 1, workspace = 0

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [TryLoad] Find-db regenerating.

/var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/build/bin/test_conv2d --float --cmode conv --pmode default --group-count 1 --disable-backward-data --disable-backward-weights --input 32, 32, 17, 17 --weights 32, 32, 1, 7 --pads_strides_dilations 0 3 1 1 1 1 --trans_output_pads 0 0 

FAILED: /var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/src/ocl/convolutionocl.cpp:783: Forward Convolution cannot be executed due to incorrect params

Forward convolution: 

Input tensor: 32, 32, 17, 17

Weights tensor: 32, 32, 1, 7

Output tensor: 

Filter: conv2d, miopenConvolution, miopenPaddingDefault, {0, 3}, {1, 1}, {1, 1}, 

MIOpen(HIP): Info [GetFindModeValueImpl] MIOPEN_FIND_MODE = HYBRID(3)

MIOpen(HIP): Info [Handle] stream: 0xa6cae40, device_id: 2

MIOpen(HIP): Info [BackwardDataGetWorkSpaceSize] 

MIOpen(HIP): Info [HipCompilerVersionImpl] 3.0.19502

MIOpen(HIP): Info [AmdRocmMetadataVersionDetect] ROCm MD version AMDHSA_COv2_COv3, MIOpen version 2.10.0.8301-71981a35

MIOpen(HIP): Info [GetBackwardSolutions] 

MIOpen(HIP): Info [Measure] Db::Prefetch time: 132.784 ms

MIOpen(HIP): Info [GetEnvFindOnlySolverImpl] 63

MIOpen(HIP): Info [ForwardGetWorkSpaceSize] 

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [BackwardWeightsGetWorkSpaceSize] 

MIOpen(HIP): Info [GetWrwSolutions] 

/var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/build/bin/test_conv2d --float --cmode conv --pmode default --group-count 1 --disable-backward-data --disable-backward-weights --input 16, 384, 8, 8 --weights 64, 384, 1, 1 --pads_strides_dilations 0 0 1 1 1 1 --trans_output_pads 0 0 

MIOpen(HIP): Info [ForwardGetWorkSpaceSize] 

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [FindConvFwdAlgorithm] requestAlgoCount = 1, workspace = 0

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [TryLoad] Find-db regenerating.

/var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/build/bin/test_conv2d --float --cmode conv --pmode default --group-count 1 --disable-backward-data --disable-backward-weights --input 16, 384, 8, 8 --weights 64, 384, 1, 1 --pads_strides_dilations 0 0 1 1 1 1 --trans_output_pads 0 0 

FAILED: /var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/src/ocl/convolutionocl.cpp:783: Forward Convolution cannot be executed due to incorrect params

Forward convolution: 

Input tensor: 16, 384, 8, 8

Weights tensor: 64, 384, 1, 1

Output tensor: 

Filter: conv2d, miopenConvolution, miopenPaddingDefault, {0, 0}, {1, 1}, {1, 1}, 

MIOpen(HIP): Info [GetFindModeValueImpl] MIOPEN_FIND_MODE = HYBRID(3)

MIOpen(HIP): Info [Handle] stream: 0xa7d6dd0, device_id: 0

MIOpen(HIP): Info [BackwardDataGetWorkSpaceSize] 

MIOpen(HIP): Info [HipCompilerVersionImpl] 3.0.19502

MIOpen(HIP): Info [AmdRocmMetadataVersionDetect] ROCm MD version AMDHSA_COv2_COv3, MIOpen version 2.10.0.8301-71981a35

MIOpen(HIP): Info [GetBackwardSolutions] 

MIOpen(HIP): Info [Measure] Db::Prefetch time: 129.584 ms

MIOpen(HIP): Info [GetEnvFindOnlySolverImpl] 66

MIOpen(HIP): Info [ForwardGetWorkSpaceSize] 

MIOpen(HIP): Info [GetForwardSolutions] 

MIOpen(HIP): Info [BackwardWeightsGetWorkSpaceSize] 

MIOpen(HIP): Info [GetWrwSolutions] 

/var/jenkins/workspace/LLibs_MIOpen_fix_v4r1bwd_tunning/build/bin/test_conv2d --float --cmode conv --pmode default --group-count 1 --disable-forward --disable-backward-data --input 64, 64, 28, 28 --weights 32, 64, 1, 1 --pads_strides_dilations 0 0 1 1 1 1 --trans_output_pads 0 0 

Memory access fault by GPU node-2 (Agent handle: 0x40ab260) on address 0x7f1783062000. Reason: Page not present or supervisor privilege.

Aborted (core dumped)

test/CMakeFiles/test_conv_igemm_dynamic_small.dir/build.make:57: recipe for target 'test/CMakeFiles/test_conv_igemm_dynamic_small' failed

make[7]: *** [test/CMakeFiles/test_conv_igemm_dynamic_small] Error 134

CMakeFiles/Makefile2:5889: recipe for target 'test/CMakeFiles/test_conv_igemm_dynamic_small.dir/all' failed

make[6]: *** [test/CMakeFiles/test_conv_igemm_dynamic_small.dir/all] Error 2

CMakeFiles/Makefile2:5896: recipe for target 'test/CMakeFiles/test_conv_igemm_dynamic_small.dir/rule' failed

make[5]: *** [test/CMakeFiles/test_conv_igemm_dynamic_small.dir/rule] Error 2

Makefile:209: recipe for target 'test/CMakeFiles/test_conv_igemm_dynamic_small.dir/rule' failed

make[4]: *** [test/CMakeFiles/test_conv_igemm_dynamic_small.dir/rule] Error 2



54/57 Test #53: test_tensor_vec ..................   Passed   14.33 sec

55/57 Test #11: test_conv2d ......................   Passed   34.08 sec

      Start 35: test_perfdb

56/57 Test #35: test_perfdb ......................***Not Run (Disabled)   0.00 sec

      Start 44: test_sqlite_perfdb

57/57 Test #44: test_sqlite_perfdb ...............***Not Run (Disabled)   0.00 sec



86% tests passed, 1 tests failed out of 7



Total Test time (real) =  34.08 sec



The following tests did not run:

	  1 - test_activation (Disabled)

	  2 - test_bn_3d_peract_test (Disabled)

	  3 - test_bn_3d_spatial_test (Disabled)

	  4 - test_bn_aux (Disabled)

	  5 - test_bn_peract_test (Disabled)

	  6 - test_bn_spatial_test (Disabled)

	  7 - test_cache (Disabled)

	  8 - test_cba_inference (Disabled)

	  9 - test_cbna_inference (Disabled)

	 12 - test_conv2d_bias (Disabled)

	 13 - test_conv3d (Disabled)

	 14 - test_conv3d_bias (Disabled)

	 15 - test_ctc (Disabled)

	 16 - test_custom_allocator (Disabled)

	 17 - test_dropout (Disabled)

	 18 - test_embed_sqlite (Disabled)

	 19 - test_find_db (Disabled)

	 20 - test_fusion_aux (Disabled)

	 21 - test_gru (Disabled)

	 22 - test_gru_dropout (Disabled)

	 23 - test_handle_test (Disabled)

	 25 - test_immed_conv3d (Disabled)

	 26 - test_include_inliner (Disabled)

	 27 - test_kernel_build_params (Disabled)

	 28 - test_lrn_test (Disabled)

	 29 - test_lstm (Disabled)

	 30 - test_lstm_dropout (Disabled)

	 31 - test_main (Disabled)

	 32 - test_mdgraph (Disabled)

	 33 - test_na_inference (Disabled)

	 34 - test_na_train (Disabled)

	 35 - test_perfdb (Disabled)

	 36 - test_pooling2d (Disabled)

	 37 - test_pooling3d (Disabled)

	 38 - test_reduce_test (Disabled)

	 39 - test_rnn_vanilla (Disabled)

	 40 - test_rnn_vanilla_dropout (Disabled)

	 41 - test_sequences (Disabled)

	 42 - test_soft_max (Disabled)

	 43 - test_solver (Disabled)

	 44 - test_sqlite_perfdb (Disabled)

	 45 - test_tensor_cast (Disabled)

	 47 - test_tensor_ops (Disabled)

	 48 - test_tensor_scale (Disabled)

	 50 - test_tensor_test (Disabled)

	 51 - test_tensor_trans (Disabled)

	 52 - test_tensor_transform (Disabled)

	 54 - test_test_errors (Disabled)

	 55 - test_type_name (Disabled)

	 56 - test_w_supertensor (Disabled)



The following tests FAILED:

	 57 - test_conv_igemm_dynamic_small (Failed)

Errors while running CTest

test/CMakeFiles/check.dir/build.make:57: recipe for target 'test/CMakeFiles/check' failed

make[3]: *** [test/CMakeFiles/check] Error 8

CMakeFiles/Makefile2:9202: recipe for target 'test/CMakeFiles/check.dir/all' failed

make[2]: *** [test/CMakeFiles/check.dir/all] Error 2

CMakeFiles/Makefile2:9209: recipe for target 'test/CMakeFiles/check.dir/rule' failed

make[1]: *** [test/CMakeFiles/check.dir/rule] Error 2

Makefile:3538: recipe for target 'check' failed

make: *** [check] Error 2

script returned exit code 2


```
</details>

**Comment by asroy** [comment]
What is the cause of issue? The test for asm-iGEMM contains some cases not supported by asm-iGEMM?

**Comment by carlushuang** [comment]
`test_conv_igemm_dynamic_small` should only apply to gfx906, not 908. But this test do not have `SKIP_UNLESS_ALL` flag and seems always keeped up in gfx908 environment. So for simplicity and easy of maintain, merge `test_conv_igemm_dynamic_small` and `test_conv_igemm_dynamic` together with `SKIP_UNLESS_ALL` set https://github.com/ROCmSoftwarePlatform/MIOpen/pull/649/files

**Comment by atamazov** [comment]
`test_conv_igemm_dynamic` and `test_conv_igemm_dynamic_small` test only FP32. These should be only used with Vega targets and only with FP32 datatypes.

This doesn't happen due to a bug in `add_custom_test`. Fix is #750

**Comment by ppanchad-amd** [comment]
@asleepzzz Please check with latest ROCm 6.0.2? If not reproducible, please close ticket. Thanks!
